app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    instructions = command.split()
+    source = instructions[1]
+    destination = instructions[2]
+
+    if len(instructions) != 3 or instructions[0] != "mv":
+        raise ValueError("Incorrect command")
+
+    if os.path.exists(destination):
+        raise ValueError("Already exists!")
+
+    path = "/".join(destination.split("/")[:-1])
```

Comment:
Instead of manually splitting and joining the path, use `os.path.join` to ensure cross-platform compatibility as per the checklist requirement: 'Use `os.path.join` to concatenate parts of the path.'